### PR TITLE
[BrokenLinksH2] Fixed broken links

### PR DESCRIPTION
**Pull request by kturetz**
Fixed broken links in iis-docs.

### CODE REVIEW

**Comment by kturetz** [comment]
**Global effort to fix broken links** 

@Rick-Anderson 
@wadepickett 

The Content & Learning team is fixing broken links on docs.microsoft.com for the rest of H2. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only link fixes and does not change other content. I’d very much appreciate your timely review and feedback on the link changes I’ve suggested. Thanks!

File: iis/extensions/configuring-application-request-routing-arr/achieving-high-availability-and-scalability-arr-and-nlb.md [hunk 1]
```diff
@@ -330,6 +330,6 @@ In this whitepaper, two main ARR scenarios were reviewed to achieve high availab
 
     - [https://technet.microsoft.com/library/cc782694.aspx](https://technet.microsoft.com/library/cc782694.aspx)
     - [https://technet.microsoft.com/library/cc778263.aspx](https://technet.microsoft.com/library/cc778263.aspx)
-    - [https://support.microsoft.com/kb/323437](https://support.microsoft.com/kb/323437)
-    - [https://support.microsoft.com/kb/890159/en-us](https://support.microsoft.com/kb/890159/en-us)
+    - `https://support.microsoft.com/kb/323437`
+    - `https://support.microsoft.com/kb/890159/en-us`
```
Comment (wadepickett) [inline iis/extensions/configuring-application-request-routing-arr/achieving-high-availability-and-scalability-arr-and-nlb.md:334]:
```suggestion
    - `https://support.microsoft.com/kb/890159`
```

**Review by wadepickett — APPROVED** [review]
I see how, and why the KB links are here they way they are.  There are not any good options.  I am leaving them as is.

File: iis/extensions/configuring-application-request-routing-arr/achieving-high-availability-and-scalability-arr-and-nlb.md (hunk 1 above)
Comment (kturetz) [reply iis/extensions/configuring-application-request-routing-arr/achieving-high-availability-and-scalability-arr-and-nlb.md]: Thanks, Wade. On rare occasions we're able to locate replacement links for the KB docs, but most of the time must comment them out.

**Pull request merged**